src/provider.ts

```diff
@@ -1,7 +1,7 @@
 import Ganache from "ganache-core";
 import { ethers } from "ethers";
 import { IConfig } from "./dataEntities/config";
-const config = require("./config.json") as IConfig;
+const config = require("./../config.json") as IConfig;
```

Comment:
also when requiring a file in upper directory, just use `../`, its just more clearer

Reply:
kk will do